source/scss/_library/_base.forms.scss

```diff
@@ -1,11 +1,39 @@
 /*------------------------------------*\
-    $SPECIFIC FORMS
+    $FORMS
 \*------------------------------------*/
 
+form ol,
+form ul {
+  list-style: none;
+  margin-left: 0;
+}
+
+legend {
+  margin-bottom: rem(6);
+  font-weight: bold;
+}
+
+fieldset {
+  border: 0;
+  padding: 0;
+  margin: 0;
+  min-width: 0;
+}
+
+input,
+select,
+textarea {
+  width: 100%;
+  border: $border-style;
+  padding: $space;
+  -webkit-appearance: none;
```

Comment:
does your build process do vendor prefixing automatically? firefox for instance won't respect the `-webkit-` prefix

Reply:
It does. I use autoprefixer. But I should remove the vendor prefix on this just to keep it all consistent.